docs/v2tov3.md

```diff
@@ -147,6 +147,71 @@ For a complete view of the GoSDK `v3`, please refer to [TODO: link API and tutor
 * Some member names changed, e.g. `MetricSet` is now `metric.Set`
 * Added `EntityMetadata` struct to fulfill the [JSON `v2` schema](#json-v2-schema-changes).
 
+## Migrate from `v2` to `v3` steps
+### Imports
+* First of all we've renamed some packages. So the first you can do when migrating from `v2` to `v3` is run these commands inside your 
+integration src folder :
+
+For **Linux**:
+```bash
+find . -type f -iname "*.go" |xargs -i {} sed -i '' 's/github.com/newrelic/infra-integrations-sdk/sdk/github.com/newrelic/infra-integrations-sdk/integration/' {}
+find . -type f -iname "*.go" |xargs -i {} sed -i '' 's/github.com/newrelic/infra-integrations-sdk/cache/github.com/newrelic/infra-integrations-sdk/persist/' {}
+find . -type f -iname "*.go" |xargs -i {} sed -i '' 's/github.com/newrelic/infra-integrations-sdk/metric/github.com/newrelic/infra-integrations-sdk/data/metric/' {}
+``` 
+
+For **MacOS**:
```

Comment:
aren't the same as Linux?

Reply:
I think that by default the `xargs` command in MacOS does not have the -i option and is replaced by -I.

And in linux -i works by default.

Reply:
Oh, I see that the -i option has been deprecated. I'll change that.
http://man7.org/linux/man-pages/man1/xargs.1.html